rest-api/model/participant_summary.py

```diff
@@ -78,6 +78,15 @@ class ParticipantSummary(Base):
 
   signUpTime = Column('sign_up_time', UTCDateTime)
 
+  # The time that this participant become a member
+  enrollmentStatusMemberTime = Column('enrollment_status_member_time', UTCDateTime)
+  # The time when we get the first stored sample
+  enrollmentStatusCoreStoredSampleTime = Column('enrollment_status_core_stored_sample_time',
```

Comment:
Keep the field name not change, because if change this to core_time, the enrollmentStatusCoreOrderedSampleTime will become more confuse.